docs/webactions.md

```diff
@@ -88,7 +88,7 @@ An OpenWhisk action that is not a web action requires both authentication and mu
 
 1. `headers`: a JSON object where the keys are header-names and the values are string values for those headers (default is no headers).
 2. `statusCode`: a valid HTTP status code (default is 200 OK).
-3. `body`: a string which is either plain text or a base64 encoded string (for binary data).
+3. `body`: a string which is either plain text or a base64 encoded string for binary data (default is empty response).
```

Comment:
With this patch? I will add an explicit test for this but a quick sniff test:

```
> curl -vk https://guest.openwhisk/default/t
*   Trying 192.168.99.100...
* Connected to guest.openwhisk (192.168.99.100) port 443 (#0)
* TLS 1.2 connection using TLS_ECDHE_RSA_WITH_RC4_128_SHA
* Server certificate: *.openwhisk
> GET /default/t HTTP/1.1
> Host: guest.openwhisk
> User-Agent: curl/7.43.0
> Accept: */*
> 
< HTTP/1.1 405 Method Not Allowed
< Server: nginx/1.11.13
< Date: Mon, 17 Apr 2017 23:54:07 GMT
< Content-Length: 0
< Connection: keep-alive
< 
* Connection #0 to host guest.openwhisk left intact
```

for this action:
```
function main() { return { statusCode: 405 } }
```